File: postit_wall/nextjs_space/query_cats.py
from prisma import Client
import asyncio

async def main():
    client = Client()
    await client.connect()
    cats = await client.category.find_many(where={'title': {'contains': 'hava', 'mode': 'insensitive'}})
    for c in cats:
        print(f"ID: {c.id}, Title: {c.title}, Slug: {c.slug}")
    await client.disconnect()

if __name__ == '__main__':
    asyncio.run(main())

File: postit_wall/nextjs_space/query_cats.mjs
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
async function main() {
  const cats = await prisma.category.findMany({ select: { name: true, ribbonColor: true, isOttActive: true } });
  console.log(cats);
}
main()
